src/libs/OptionsListUtils.ts

```diff
@@ -1741,7 +1742,10 @@ function getOptions(
     }
 
     // Exclude the current user from the personal details list
-    const optionsToExclude: Option[] = [{login: currentUserLogin}, {login: CONST.EMAIL.NOTIFICATIONS}];
+    const optionsToExclude: Option[] = [{login: CONST.EMAIL.NOTIFICATIONS}];
+    if (!includeSelfDM) {
+        optionsToExclude.push({login: currentUserLogin});
+    }
```

Comment:
@s77rt Using like this leads to the jest test 2 is failed

Reply:
If I revert that change and update [this](https://github.com/Expensify/App/blob/76ca7814304e143c86d4a2031c69af8aba8e17a9/src/libs/OptionsListUtils.ts#L1774) to:
```
 if (!reportOption.isSelfDM &&
                (!includeThreads &&
                    (!!reportOption.login || reportOption.reportID) &&
                    optionsToExclude.some((option) => option.login === reportOption.login || option.reportID === reportOption.reportID))
            ) {
```
the test is passed

Reply:
I'm not seeing any failing test

Reply:
Here is the test result https://github.com/Expensify/App/actions/runs/8780426190/job/24090318554?pr=40672. can you check it?